changelog.md

```diff
@@ -19,6 +19,9 @@ Saves from 4.0.0 are compatible with 4.1.0.
 * **[UI]** Added ability to take notes and have those notes appear as a kneeboard page.
 * **[UI]** Hovering over the weather information now dispalys the cloud base (meters and feet).
 * **[UI]** Google search link added to unit information when there is no information provided.
+* **[UI]** Control point name displayed with ground object group name on map.* 
```

Comment:
?

Reply:
Came in with the master merge, not written by me. Deleted it.